_rules/element-lang-valid-de46e4.md

```diff
@@ -141,22 +186,52 @@ The rule applies to elements within the `body` of a webpage. `html` elements are
 
 ```html
 <html lang="en">
-	<body></body>
+	<body>
+		They wandered into a strange Tiki bar on the edge of the small beach town.
+	</body>
 </html>
 ```
 
 #### Inapplicable Example 2
 
-An empty value for the `lang` attribute is ignored, as the rule only applies to `lang` attributes that are not empty ("").
+An empty `lang` [attribute value][] is ignored, as the rule only applies to `lang` attributes that are not empty (`""`).
+
+```html
+<html>
+	<body>
+		<article lang="">
+			They wandered into a strange Tiki bar on the edge of the small beach town.
+		</article>
+	</body>
+</html>
+```
+
+#### Inapplicable Example 3
+
+The `lang` [attribute value][] does not have a valid language tag, and does not have a [visible][] [text node][] as a [descendant][] in the [flat tree][].
 
 ```html
 <html>
 	<body>
-		<article lang=""></article>
+		<p lang="hidden">
```

Comment:
Use a valid language. Having an invalid one does not bring anything to that example (it's about visible/accessible text).
```suggestion
		<p lang="en">
```

Reply:
Right but then this does not prove the case this test is for; that an invalid lang attribute should not fail if it's on a hidden element.